sklearn/preprocessing/data.py

```diff
@@ -2537,7 +2537,7 @@ class PowerTransformer(BaseEstimator, TransformerMixin):
     >>> print(pt.fit(data))
     PowerTransformer(copy=True, method='yeo-johnson', standardize=True)
     >>> print(pt.lambdas_)
-    [1.38668178e+00 5.93926346e-09]
+    [ 1.38668178 -3.10053332]
```

Comment:
Can you clarify what change has affected this? The bounds checking?

Reply:
Yes definitely the bound checking.
The actual transformed values are the same as before.

Also, yeo-johnson transformed plots from `plot_map_data_to_normal.py` look better:

![yeo](https://user-images.githubusercontent.com/1190450/48025825-74339a80-e112-11e8-9fb4-d64624252526.png)

Reply:
> The actual transformed values are the same as before.

How is that possible if the lambda changes?

Reply:
That's because of `standardize=True`.

I get different values on master and on this branch (as expected) when it's False